jwql/website/apps/jwql/views.py

```diff
@@ -145,8 +145,9 @@ def about(request):
     return render(request, template, context)
 
 
-@auth_required
-def archived_proposals(request, user, inst):
+# @auth_required
```

Comment:
@tnking97 This is great for getting things to work temporarily, but we should probably switch back to these commented-out parts before we merge this PR, since it has to do with #558 and not so much this PR.